cupy/linalg/norms.py

```diff
@@ -57,7 +54,7 @@ def norm(x, ord=None, axis=None, keepdims=False):
             return abs(x).min(axis=axis, keepdims=keepdims)
         elif ord == 0:
             # Zero norm
-            return (x != 0).sum(axis=axis, keepdims=keepdims, dtype=x.dtype)
+            return (x != 0).sum(axis=axis, keepdims=keepdims, dtype='d')
```

Comment:
I think it is better to add a comment so that some developers do not mistakenly think that it is a bug. How about `Convert to Python float in accordance with NumPy`? (correct me if the description of dtype is not appropriate)